### ISSUE TITLE
Compatibility with Opensim 4.0

### ISSUE THREAD

**Issue by XV25** [RealTimeBiomechanics/rtosim#18]
Hi,

Could you tell me if this package is compatible with Opensim 4.0, please?

Regards,

**Comment by cpizzolato** [comment]
no, it's not. I am working on an update to opensim 4.2, but it's not going to come for a few weeks

**Comment by RobertoRoos** [comment]
My group is using RTOSim for an ongoing project and we modified it to work with OpenSim 4.3. Would you be interested in merging a pull request?

If you are not interested in maintaining this (which I could understand) we'll keep our own fork instead.

**Comment by cpizzolato** [comment]
That would be fantastic, I am very interested. Your contribution is much appreciated!

**Comment by RobertoRoos** [comment]
Alright, trying to tidy up our version now.

Do you want to maintain OpenSim 4.0? There have been some breaking changes in 4.3, so we can either:

- Make the master branch compatible with 4.3 (and stop updating for 4.0)
- Split branches and maintain RTOSIM versions for 4.0 and 4.3 (will be some work to update both)
- Try our best to make the code compatible with either (will result in lot's of compiler commands and ugly code)

Which has your preference?
